test/guip.jl

```diff
@@ -1,11 +1,11 @@
-using Bridge, FixedSizeArrays, Distributions
+using Bridge, StaticArrays, Distributions
 using Base.Test
 #import Bridge: b, σ, a, transitionprob
-const percentile = 2.3 
+const percentile = 100.3 
 
```

Comment:
what would be a good procedure for probabilistic tests? would it be ok for the test to fail with a small probability?